pages/index.style.tsx

```diff
@@ -5,7 +5,7 @@ const Title = styled('h1')({
   margin: '0',
   lineHeight: '1.15',
   fontSize: '4rem',
-
+  fontFamily: "Varela Round, sans-serif;",
```

Comment:
Maybe we could create a folder for example global-style and add there files for global use. For example, colors file, breakpoints file, fonts file, space file etc